cmake/modules/RootBuildOptions.cmake

```diff
@@ -65,7 +65,7 @@ ROOT_BUILD_OPTION(afs OFF "AFS support, requires AFS libs and objects")
 ROOT_BUILD_OPTION(alien OFF "AliEn support, requires libgapiUI from ALICE")
 ROOT_BUILD_OPTION(asimage ON "Image processing support, requires libAfterImage")
 ROOT_BUILD_OPTION(astiff ON "Include tiff support in image processing")
-ROOT_BUILD_OPTION(bonjour ON "Bonjour support, requires libdns_sd and/or Avahi")
+ROOT_BUILD_OPTION(bonjour OFF "Bonjour support, requires libdns_sd and/or Avahi")
 ROOT_BUILD_OPTION(builtin_afterimage ON "Build included libAfterImage, or use system libAfterImage")
```

Comment:
I would love for this to default to `OFF` too, so ROOT can find a system version of libafterimage by default if it's there.

Reply:
I have no strong opinion here. I am not sure of the reason why this setting is what it is...

Reply:
Because ROOT's AfterImage has fixes on top of the default one. (IIRC Guilherme was part of the discussion about a year or two ago, with Olivier and Matevz?). I.e. we should prefer the built-in one.

Reply:
Yeah, the fixes in ROOT are from the system version of libAfterImage in Gentoo, so I would like to prefer the system one. Unless you have a really old distro, in which case it doesn't matter, if you have a system version of libAfterImage, it should be usable with ROOT without problems or needing any extra patches. The fixes were for allowing libAfterImage to compile against newer versions of giflib, so if you already compiled libAfterImage, there's no need to use the builtin one from ROOT.

Reply:
Gentoo actually has more fixes than ROOT on the system libAfterImage, BTW.

Reply:
No that's not all there is, IIRC we have more patches.

Reply:
My change (porting a Gentoo patch) is the last one. The only other useful change seems to be a minor adaptation of the build system for ARM. The change before that is just some typo and gender neutral language fixes in comments...